flow-plugins/flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/Reflector.java

```diff
@@ -38,8 +38,9 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import tools.jackson.core.JacksonException;
+import tools.jackson.core.JacksonException;
```

Comment:
import seems duplicated
```suggestion
import tools.jackson.core.JacksonException;
```

Reply:
Removed. Probably duplicated by mistake from one of the multiple merges done.